### PR TITLE
Rservice docs

### PR DESCRIPTION
**Pull request by h00die**
Adds docs for the rservices. yes those ones from the 1980s.
To answer the question before its asked, its 2017.

Part of #8296 

## Verification

- [x] Check my spelling and grammar and links

### CODE REVIEW

**Comment by wvu-r7** [comment]
What year... oh.

File: documentation/modules/auxiliary/scanner/rservices/rsh_login.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+## Vulnerable Application
+
+The R Services (rexecd, rlogind, and rshd) are a suite of unencrypted remote command/login services developed in the 1980s.
+These services are all but unused in modern computing, as they have been replace by telnet and ssh.
+
+`rsh` relies on host names as a security mechanism.  Utilizing `+` can wildcard so any computer can connect.  In the following
```
Comment (busterb) [inline documentation/modules/auxiliary/scanner/rservices/rsh_login.md]: you might note there are exceptions to this rule in proprietary Unix systems. For instance, Kerberos is also supported by the rsh in AIX (though the contents of the session are still unencrypted):

https://www.ibm.com/support/knowledgecenter/en/ssw_aix_71/com.ibm.aix.cmds4/rsh.htm

Useful info from a pentester PoV, where you might get access to an AIX LPAR and pivot around.

**Comment by h00die** [comment]
:muscle: your AIX knowledge!!!

This is part of why im actually really enjoying writing the docs, everyone has great insight (@wvu-r7 got me on x11 stuff).

**Comment by wvu-r7** [comment]
Thank you for doing the real work, @h00die. 🙇

**Comment by busterb** [comment]
Thanks @hoodie.

**Comment by busterb** [comment]
## Release Notes

This adds documentation on rservice (rsh, rlogin, rexec) modules.

**Pull request merged**